apps/src/code-studio/reporting.js

```diff
@@ -39,13 +40,173 @@ reporting.getLastServerResponse = function () {
   return lastServerResponse;
 };
 
+/**
+ * Validate that the provided field on our report object is one of the given
+ * type
+ * @param {string} key
+ * @param {*} value
+ * @param {string} type
+ */
+function validateType(key, value, type) {
+  let typeIsValid = false;
+  if (type === 'array') {
+    typeIsValid = typeIsValid || _.isArray(value);
```

Comment:
Why not [`Array.isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)?